### PR TITLE
docs(material/stepper): misleading default icon documentation

### LINKED ISSUE #22498: docs-bug(stepper): misleading default icon documentation
#### Documentation Feedback

The stepper documentation says the default icons are 
```
`create` and `done`
```
but it seems to be
```
`edit` and `done`
```

**Affected documentation page:**
* https://material.angular.io/components/stepper/overview#overriding-icons
* [src/material/stepper/stepper.md](https://github.com/angular/components/blob/master/src/material/stepper/stepper.md)

### PR DESCRIPTION
**Pull request by oberbeck**
Fixes the default stepper icon documentation.

Fixes #22498

### CODE REVIEW

**Comment by oberbeck** [comment]
@googlebot I signed it!

File: src/material/stepper/stepper.md [hunk 1]
```diff
@@ -123,7 +123,7 @@ linear stepper) and the user has interacted with the step. The user, however, ca
 this default `completed` behavior by setting the `completed` attribute as needed.
 
 #### Overriding icons
-By default, the step headers will use the `create` and `done` icons from the Material design icon
+By default, the step headers will use the `edit` and `done` icons from the Material design icon
```
Comment (crisbeto) [inline src/material/stepper/stepper.md:126]: I think that the current text is correct. See https://github.com/angular/components/blob/master/src/material/stepper/step-header.ts#L139

Comment (oberbeck) [reply src/material/stepper/stepper.md:126]: Please check out the following example where I am overwriting the `edit` as well as the `create` icons with the specific text. For the active, untouched step, the edit icon is used:
https://stackblitz.com/edit/angular-q4sfkg?file=src/app/stepper-states-example.html

```
  <ng-template matStepperIcon="edit">
    <span>EDIT</span>
  </ng-template>
  <ng-template matStepperIcon="create">
    <span>CREATE</span>
  </ng-template>
```

Comment (crisbeto) [reply src/material/stepper/stepper.md:126]: The confusion here comes from the fact that the step state is `edit`, but the icon it is using to represent it is called `create` so I think that the docs are correct. In your example, when you set `matStepperIcon="edit"`, it means that the particular icon will be used for the `edit` state, not that it'll use an icon called `edit`.

Comment (oberbeck) [reply src/material/stepper/stepper.md:126]: I see; in that case it is misleading; as the documentation clearly states, by default the icons are used as states:

https://github.com/angular/components/blame/master/src/material/stepper/stepper.md#L138

**Comment by crisbeto** [comment]
Closing since there hasn't been any activity in a while.

**Pull request closed without merging**